apps/planning-suite-kit/contracts/PlanningKit.sol

```diff
@@ -91,12 +91,13 @@ contract PlanningKit is KitBase {
         MiniMeToken token = tokenFactory.createCloneToken(MiniMeToken(0), 0, "Autark Token", 0, "autark", true);
         // token.changeController(tokenManager);
         // token.generateTokens(root, 1);
+        token.generateTokens(address(0xb4124cEB3451635DAcedd11767f004d8a28c6eE7), 1 ether);
```

Comment:
I completely agree, I just relied on the authority code here to prevent doubts, in other words, your previous [branch-cleanup code](https://github.com/spacedecentral/planning-suite/blob/ec64ffb31777b23a56d5955a7513671ab9cbecbb/apps/planning-suite-kit/contracts/PlanningKit.sol#L111). I will try with the line above that it should work I guess...